web/impact/impact/v1/helpers/judge_criterion_helper.py

```diff
@@ -1,25 +1,31 @@
 # MIT License
 # Copyright (c) 2017 MassChallenge, Inc.
 
+from django.db.models import F
 from impact.v1.helpers.criterion_helper import CriterionHelper
 
 
 class JudgeCriterionHelper(CriterionHelper):
-    '''Helper for JudgeCriteria. JudgeCriteria specify a feature which 
+    '''Helper for JudgeCriteria. JudgeCriteria specify a feature which
     all applications are "looking for", for example "All startups should be
     read by at least one female judge".
     JudgeCriteria require one CriterionOptionSpec for each feature sought, for
-    example, "one female judge and one male judge" would require two 
+    example, "one female judge and one male judge" would require two
     OptionSpecs. Weight and count can be set independently for each option.
     '''
 
     def option_for_field(self, field):
         return field
 
     def judge_matches_option(self, judge_data, option):
-        return option == judge_data.get(self.cache_judge_field)
```

Comment:
There has been a bit of a misunderstanding, coz I'm lost. I can understand the logic and all but the consistency is not right I think.

This field, the `cache_judge_field` was created for a single use-case, that is when querying data from the `JudgeDataCache`. Now the new field `cache_key` is being used across the logic as the key used on the `populate_criterion_total_capacities_cache`.

Reply:
The problem was that having two different names for the entry in the cache caused the judge_matches_option check to fail sometimes. So the fix was to use the same name throughout. 
I feel like there's more refactor to get to here, the question is, is now the right time to do it?

Reply:
Okay, we could leave this as is but I'll put comments elsewhere so that were not duplicating some of our logic